js/sites.js

```diff
@@ -72,11 +72,12 @@
         aIndex = this.cell.index;
       }
 
-      gNewTab.sendToBrowser("NewTab:PinLink", {
+      gNewTab.sendToBrowser("NewTab:GetUpdatePages", {
         link: this._link,
```

Comment:
we're sending the link and the index up but on AboutNewTab.jsm's side we don't actually make use of it. Since we're going directly to update pages we don't need to send any data up anymore. See: https://github.com/mozilla/newtab-dev/blob/906f24ce3ff84233c7d3bba07611374e290a2f25/browser/components/newtab/AboutNewTab.jsm#L270-L278, we're not using the data from the message anyways